static/index.js

```diff
@@ -149,19 +149,30 @@ window.addEventListener('load', e => {
   setupNavigationSection();
 });
 
-const setupBadgeSection = () => {
-  const badgeSection = document.querySelector('#badge_section');
-  const badgeAPI = window.Badge || window.ExperimentalBadge;
+function setBadge(...args) {
+  if (navigator.setExperimentalAppBadge)
+    navigator.setExperimentalAppBadge.apply(navigator, args);
+  else if (window.ExperimentalBadge)
+    ExperimentalBadge.set.apply(null, args);
+}
 
-  if (!badgeSection) {
-    return;
-  }
-  // Check that the API is available.
-  if (!badgeAPI) {
-    console.log("Badge API not detected :'(");
+function clearBadge() {
+  if (navigator.clearExperimentalAppBadge)
+    navigator.clearExperimentalAppBadge();
+  else if (window.ExperimentalBadge)
+    ExperimentalBadge.clear();
+}
+
+function badgeAPIAvailable() {
+  return !!(navigator.setExperimentalAppBadge || window.ExperimentalBadge);
+}
```

Comment:
Below, you use this as a constant, not a function, which is probably more sensible as it can't change.

```suggestion
const badgeAPIAvailable = navigator.setExperimentalBadge || window.ExperimentalBadge;
```

Optional Nit: no need for the !!, you're treating this as a boolean.

Reply:
Done as a constant. But I do want to keep the !! because it's confusing that `badgeAPIAvailable` would be a function, or an interface object.